static/js/services/playsound.js

```diff
@@ -22,141 +22,148 @@
 "use strict";
 define(['underscore', 'Howler', 'require'], function(_, Howler, require) {
 
-	var SoundInterval = function(sound, id, time) {
-		this.sound = sound;
-		this.id = id;
-		this.interval = null;
-		this.time = time;
-	};
-	SoundInterval.prototype.start = function() {
-		if (this.interval !== null) {
-			return;
-		}
-		var id = this.id;
-		var player = _.bind(function() {
-			return this.sound.play(id);
-		}, this);
-		player();
-		this.interval = setInterval(player, this.time);
-	};
-	SoundInterval.prototype.stop = function() {
-		clearInterval(this.interval);
-		this.interval = null;
-		delete this.sound.intervals[this.id];
-	};
-
-	var Sound = function(options, aliases) {
-
-		this.sound = null;
-		this.intervals = {};
-		if (options) {
-			this.initialize(options, aliases);
-		}
+	// playSound
+	return ["appData", function(appData) {
 
-	};
+		var SoundInterval = function(sound, id, time) {
+			this.sound = sound;
+			this.id = id;
+			this.interval = null;
+			this.time = time;
+		};
+		SoundInterval.prototype.start = function() {
+			if (this.interval !== null) {
+				return;
+			}
+			var id = this.id;
+			var player = _.bind(function() {
+				return this.sound.play(id);
+			}, this);
+			player();
+			this.interval = setInterval(player, this.time);
+		};
+		SoundInterval.prototype.stop = function() {
+			clearInterval(this.interval);
+			this.interval = null;
+			delete this.sound.intervals[this.id];
+		};
 
-	Sound.prototype.initialize = function(options, aliases) {
+		var Sound = function(options, aliases) {
```

Comment:
My file has only tabs. Because the only thing that changed on this line was the tab and `var` git is emphasizing this for whatever reason. A bit confusing at the first glance.